libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -58,7 +58,7 @@ public class OAuthPrompt : Dialog
         private const string PersistedExpires = "expires";
 
         // Default prompt timeout of 15 minutes (in ms)
-        private const int DefaultPromptTimeout = 900000;
+        private const int DefaultPromptTimeout = TurnStateConstants.OAuthLoginTimeoutMsValue;
```

Comment:
Making timeouts of type TimeSpan makes them more readable btw, i.e. TimeSpan.FromMinutes(15) is way more readable than 900000 milliseconds (i.e. I have to do math to understand how long it is)

Reply:
Yes, I had removed this from here, and where I put it I changed it to a TimeStamp.